Legion/VaultOfTheWardens/Tormentorum.lua

```diff
@@ -115,3 +114,71 @@ end
 function mod:FrighteningShout(args)
 	self:Message(args.spellId, "Urgent", "Warning", CL.casting:format(args.spellName))
 end
+
+do
+	local prev = 0
+	function mod:ShadowCrashDamage(args)
+		local t = GetTime()
+		if self:Me(args.destGUID) and t-prev > 1.5 then
+			prev = t
+			self:Message(args.spellId, "Personal", "Alert", CL.underyou:format(args.spellName))
+		end
+	end
+end
+
+do
+	local scheduled = false
+
+	local function sortTable(x, y)
+		if x.stacks == y.stacks then
+			return x.name > y.name
+		else
+			return x.stacks > y.stacks
+		end
+	end
+
+	local function printStacks(self, spellId)
+		local meOnly = self:CheckOption(spellId, "ME_ONLY")
+		local spellName = self:SpellName(spellId)
+		if meOnly then
+			local stacks = select(4, UnitDebuff("player", spellName)) or 0
+			if stacks > 6 then
+				self:StackMessage(spellId, self:UnitName("player"), stacks, "Urgent", stacks < 9 and "Alarm" or "Warning")
+			end
+		else
+			local stacksOnMe = 0
+			local affectedPlayers = {}
+			for unit in self:IterateGroup() do
+				local stacks = select(4, UnitDebuff(unit, spellName)) or 0
+				if stacks > 6 then
+					affectedPlayers[#affectedPlayers + 1] = { name = self:UnitName(unit), stacks = stacks }
+					if UnitIsUnit("player", unit) then
+						stacksOnMe = stacks
+					end
+				end
+			end
+			if #affectedPlayers == 0 then
+				scheduled = false

```

Comment:
You can just set `scheduled = nil` right at the start of the function, no need to think about it further then ;)